pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -129,6 +131,8 @@ func (lvm *pmemLvm) CreateDevice(name string, size uint64, nsmode string) error
 }
 
 func (lvm *pmemLvm) DeleteDevice(name string, flush bool) error {
+	devicemutex.Lock()
+	defer devicemutex.Unlock()
```

Comment:
The locking should be on device `name` I guess. Deleting two different devices parallelly shouldn't be problem.

Reply:
Delete or create tries to change single system in host, also in LVM mode.
I think it adds safety if we avoid change requests from parallel threads at same time.
Introducing another set of locks would make the code more complex,
and there is no perf. requirement to achieve high speeds here, so my idea is to play safe.

Reply:
I agree that it is better to play it safe here.